### ISSUE TITLE
GM_notification timeout problem

### ISSUE THREAD

**Issue by FabianoLothor** [Tampermonkey/tampermonkey#704]
Using Chrome:

Timeout `30000` hidding with approximately 7 seconds.

Code: 

 ```javascript
           GM_notification ({
                text: 'Test notification body.',
                title: 'Test notice title',
                //highlight: true,
                timeout: 30000,
                onclick: function () {
                    console.log("Notice clicked.");
                    window.focus();
                }
            });
```

**Comment by derjanb** [comment]
What is your OS version?

**Comment by FabianoLothor** [comment]
`Version 74.0.3729.131 (Official Build) (64-bit)
`

**Comment by derjanb** [comment]
What is your Windows, MacOS or Linux version?

**Comment by FabianoLothor** [comment]
`windows `

![image](https://user-images.githubusercontent.com/2192995/57399304-5de0c700-71a7-11e9-8d7f-76f1835f3306.png)

**Comment by FabianoLothor** [comment]
Just for test:

```
// ==UserScript==
// @name         foo
// @namespace    http://fabianolothor.com.br/
// @version      1.0.0
// @description  Global
// @author       Fabiano (Lothor)
// @match        *://*/*
// @grant        GM_notification
// @grant        GM_getValue
// @grant        GM_setValue
// @grant        GM_deleteValue
// ==/UserScript==
(function() {
    GM_notification({text: "this is a test", timeout: 30000});
})();
```

**Comment by FabianoLothor** [comment]
Is it some setting on my Chrome or TamperMonkey?

**Comment by FabianoLothor** [comment]
I make a test in a linux machine and the problem is the same.

**Comment by FabianoLothor** [comment]
You can add this parameter: https://web-push-book.gauntface.com/chapter-05/03-notification-behaviour/#requires-interaction

`requires-interaction` until the timeout ended.

**Comment by FabianoLothor** [comment]
`// @require      https://pushjs.org/scripts/push.min.js`

I solve my problem for now: https://pushjs.org/

**Comment by derjanb** [comment]
Fixed at TM BETA 4.9.5960. "requireInteraction" is now set where ever possible.

**Comment by fjleon1980** [comment]
since the new tampermonkey update on november 20 all my scripts notifications never close on their own anymore. i tried adding the timeout parameter but it is ignored. i want the previous behavior of notifications closing on their own, no matter what i do
